tests/boards/ehl_crb/gpio/src/main.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @file
+ * @brief Sample Application for GPIO driver.
+ * This sample shows how to configure gpio pins to output direction to
+ * control pin levels, and also how to configure pins to input direction
+ * to trigger interrupts.
+ * The sample uses 2 pins, one as output pin, and the other
+ * as input pin. Connect these 2 pins, output pin will toggle the level,
+ * this would trigger interrupts for the input pin.
+ *
+ * @brief Hardware setup.
+ * Connect pins 13 and 17 of J1E2 header of EHL CRB board in loopback.
```

Comment:
Can you please help us understand the relationship for the pin numberings across these files? We see in prj.conf that it is `GPIO_COM_0_B pin 28 is used as output and pin 27 as input pin`, and here we see 13 and 17, then in a few lines we see macros for GPIO_* pins 23 and 4. This could be tricky for a user to set up correctly on their HW pins as a sample?

Reply:
Pin numbers are 28 and 27 and they map to pin 13 and 17 on the board. 23 and 4 is the pin number in the community to which the pin belongs to.

Reply:
Pins used will be 23 and 4. They map to pin 13 and 17 on the EHL CRB board.